src/components/shared/browser-docs-wip/browser-docs-wip.view.js

```diff
@@ -2,10 +2,10 @@ import Blockquote from 'components/shared/blockquote';
 import React from 'react';
 
 const BrowserDocsWIP = () => (
-  <Blockquote mod="warning">
+  <Blockquote mod="attention">
     <p>
-      This documentation is a work in progress, and some of the functionality
-      that is listed below may not behave as expected.
+      The xk6-browser API is a work in progress. Some of the following
```

Comment:
Why do you need to specify `xk6-browser`? Couldn't we just say:

```suggestion
      The API is a work in progress. Some of the following
```

Reply:
Yeah, that's a good point.  This would make the componant more generally usable. The only thing I worry about is that "The API" could get confused in the reader's mind as "The entire k6 API". Unlikely, but to be a little more specific, how about "this"

> This API is a ...

Reply:
Sounds good to me 👍